app/assets/stylesheets/themes/dark.css

```diff
@@ -225,39 +191,94 @@
   ***********************************************/
 
   /* Buttons: Default */
-  --btn-color-hover: var(--accent-brand-lighter);
+  --btn-bg: transparent;
```

Comment:
A lot of this code has been actually copy'pasted from light theme and it could be avoided but I think that's fine for now. As long as we tweak little things here I'd consider we're still in "transition" stage between old variables and new variables so having it all here makes it easier to make changes 🤷